applications/vessel-history/src/features/profile/components/activity/Activity.tsx

```diff
@@ -35,6 +40,21 @@ const Activity: React.FC<ActivityProps> = (props): React.ReactElement => {
   }, [])
   const closeModal = useCallback(() => setIsOpen(false), [])
 
+  const {  setMapCoordinates } = useViewport()
+  const { dispatchQueryParams } = useLocationConnect()
+  const selectEventOnMap = useCallback((event: RenderedEvent) => {
+    dispatch(setHighlightedEvent({id: event.id} as ApiEvent))
+    dispatchQueryParams({ latitude: event.position.lat, longitude: event.position.lon })
```

Comment:
If you call `setMapCoordinates` in the next line then dispatching the query params in this one should not be necessary, as far as I understood it's already done by `setMapCoordinates`

Reply:
I will see